src/sections/NoVotingSection.tsx

```diff
@@ -1,17 +1,24 @@
-import styles from "./NoVotingSection.module.scss";
-import sofa from "../assets/sofa.png";
+import {CouchPanel} from "../components/CouchPanel";
 
 export interface NoVotingSectionProps {
-  error?: BallotError;
+  type?: BallotError;
 }
 
 export type BallotError = "NO_RUNNING_BALLOT" | "SERVER_ERROR";
 
 export const NoVotingSection = ({
-  error = "NO_RUNNING_BALLOT",
-}: NoVotingSectionProps) => {
-  const getErrorText = (error: BallotError): string => {
-    switch (error) {
+                                  type = "NO_RUNNING_BALLOT",
+                                }: NoVotingSectionProps) => {
```

Comment:
I recommend using a formatter such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to have consistent formatting across the code base and minimize unnecessary merge conflicts due to inconsistent formatting. Also, it's a good idea to enable "format on save" in your IDE/editor, so you don't forget to format your code.

Reply:
The thing is, we have a Prettier Definition but somehow it is not taking affect in IntelliJ

Reply:
I don't use IntelliJ because I have the Ultimate version, but this article might be helpful: https://www.jetbrains.com/help/idea/prettier.html

Reply:
@LeoTuet @94726

Reply:
Yeah, for some reason I couldn't install Prettier in IntellJ. Just updated to a new version which seems to have fixed it.